tests/python/test_fuse_dynamic.py

```diff
@@ -0,0 +1,58 @@
+import taichi as ti
+import time
+import pytest
+
+
+def benchmark_fuse_dynamic_x2y2z(size=1024**2, repeat=10, first_n=100):
```

Comment:
Why `benchmark`?

Reply:
Eventually, we should set up benchmarks and/or regression tests using this function as well.